static/app/components/sidebar/index.tsx

```diff
@@ -276,6 +278,17 @@ function Sidebar() {
                   {...HTTPModuleBadgeProps}
                 />
               </Feature>
+              <Feature features="performance-cache-view" organization={organization}>
+                <SidebarItem
+                  {...sidebarItemProps}
+                  label={
+                    <GuideAnchor target="performance-cache">{t('Cache')}</GuideAnchor>
```

Comment:
Got bitten _twice_ by this already, so going to recommend you set this in `settings.ts`!

Reply:
👍